File: Python/A1/A1-019.py
# Author: Korawit Suporn
# Date  : 23 Apr. 2025
# Editor: Vim

arr = [int(input()) for _ in range(3)]

if sum(arr) / arr[0] == 3:
    print("all the same")
elif arr[0] == arr[1] or arr[1] == arr[2] or arr[0] == arr[2]:
    print("neither")
else:
    print("all different")

File: C.C++/A1-019.cpp
#include <iostream>
using namespace std;

int main()
{
    int n[4]; // use int for numbers

    // input
    cin>> n[0]>> n[1]>> n[2];

    // process
    if (n[0] == n[1] && n[1] == n[2])
    {
        // output
        cout<< "all the same";
    }
    else if (n[0] != n[1] && n[1] != n[2] && n[0] != n[2])
    {
        cout<< "all different";
    }
    else
    {
        cout<< "neither";
    }
}